core/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for core project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-lmt=s86+%deh+!!+i93mf6^3u&2yknykdm=bhhlf*jcag4@=uz"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
```

Comment:
The `ALLOWED_HOSTS` list is empty. In a production environment, you should specify the domain names or IP addresses that your Django site can serve. This is a security measure to prevent HTTP Host header attacks.